pkg/controller/direct/bigquery/table_mapper.go

```diff
@@ -333,12 +374,29 @@ func CsvOptions_ToProto(mapCtx *direct.MapContext, in *krmv1beta1.CsvOptions) *p
 		return nil
 	}
 	out := &pb.CsvOptions{}
-	out.FieldDelimiter = direct.ValueOf(in.FieldDelimiter)
-	out.SkipLeadingRows = direct.ValueOf(in.SkipLeadingRows)
+	// if spec.externalDataConfiguration.csvOptions.fieldDelimiter is not set. Default to `,`.
+	if in.FieldDelimiter == nil {
```

Comment:
Nit: I think defaulting behaviors you added in the mapper should probably be extracted out into a separate defaults file like https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/64ed30faf50512cc4a0947c1f5291de44ca706cb/pkg/controller/direct/sql/sqlinstance_defaults.go.